src/createRouter.ts

```diff
@@ -173,8 +173,8 @@ const createRouter: RouterFactory = compose<RouterMixin, RouterOptions>({
 			path,
 			cancel,
 			defer () {
-				let cancel: () => void;
-				let resume: () => void;
+				let cancel: () => void = () => {};
+				let resume: () => void = () => {};
```

Comment:
Do you think there's any benefit to declaring a `noop` constant which can then be used here?